SDK/MobileCenter/Microsoft.Azure.Mobile.iOS/MobileCenter.cs

```diff
@@ -163,7 +163,7 @@ public static bool Enabled
         /// <remarks>
         /// The identifier is lost if clearing application data or uninstalling application.
         /// </remarks>
-        public static Guid? InstallId => Guid.Parse(iOSMobileCenter.InstallId().AsString());
+        public static Guid? InstallId => Guid.Parse(iOSMobileCenter.InstallId().ToString());
```

Comment:
But you fixed a crash on iOS 8 by using AsString instead of ToString(), why reverting it?